storybookjs/storybook
# Remove parent from useExpanded, check for parent filtered state

Issue:

## What I did

@ndelangen and I worked on setting parents and filtered state

## How to test

- Is this testable with Jest or Chromatic screenshots?

Yes, should see no changes. 

Should be to untoggle currently selected parent folder

- Does this need a new example in the kitchen sink apps?

no
- Does this need an update to the documentation?

no 

If your answer is yes to any of these, please make sure to include it in your PR.
